bin/interop-encode.c

```diff
@@ -319,7 +319,7 @@ main (int argc, char **argv)
         }
     }
 
-    if (0 != lsqpack_enc_init(&encoder, stderr,  dyn_table_size,
+    if (0 != lsqpack_enc_init(&encoder, NULL,  dyn_table_size,
```

Comment:
Is this change related?

Reply:
It is not related but running the tests takes ~5 minutes with logging enabled.

Reply:
Maybe add a flag to run in "coverage" mode and have it off by default?  Because as a standalone tool, it's mostly useful for debugging and that's where logs are useful.

Reply:
I see. Can I match the pattern for _interop-decode.c_ where it can be specified by command line and is off for testing?

Reply:
Sure.